examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>We will need a better model for a good example...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>This is only trained with 150 samples for testing. In the final version will be trained with the whole dataset and reaches about 80% accuracyy.</p>